data_steward/constants/validation/participants/readers.py

```diff
@@ -1,46 +1,41 @@
 from constants.validation.participants import identity_match as id_match
 
 # Select observation table attributes to validate
-PPI_OBSERVATION_VALUES = (
-    'SELECT person_id, observation_source_concept_id, value_as_string '
-    'FROM `{project}.{dataset}.{table}` '
-    'WHERE observation_source_concept_id={field_value} '
-    'ORDER BY person_id'
-)
+PPI_OBSERVATION_VALUES = ('SELECT person_id, observation_source_concept_id, value_as_string '
+                          'FROM `{project}.{dataset}.{table}` '
+                          'WHERE observation_source_concept_id={field_value} '
+                          'ORDER BY person_id')
 
 # Select values from ehr person table
-EHR_PERSON_VALUES = (
```

Comment:
nit:  So, this is a difference in the google formater and the pep-8 formater.  With the google style, lines 12 - 15 
```
EHR_PERSON_VALUES = (
    'SELECT person_id, {field} '
    'FROM `{project}.{dataset}.{table}` '
)
```
become line 10, like below.  
```
EHR_PERSON_VALUES = ('SELECT person_id, {field} ' 'FROM `{project}.{dataset}.{table}` ')
```
I personally prefer to have the query separated on multiple lines because it's a little easier to read "formatted SQL".  And it didn't join the two strings like a dev would have.  

This is the pep-8 version:
```
EHR_PERSON_VALUES = ('SELECT person_id, {field} '
                     'FROM `{project}.{dataset}.{table}` ')
```
This seems to be a rare occurrence, it just caught my eye.